crates/autopilot/src/arguments.rs

```diff
@@ -133,6 +133,10 @@ pub struct Arguments {
 
     #[clap(long, env, default_value = "0")]
     pub limit_order_price_factor: f64,
+
+    // Enable background quoting for limit orders.
+    #[clap(long, env, default_value = "true")]
```

Comment:
I believe because `bool` arguments are treated a bit differently by `clap` we can not provide `true` as the default value here. If `true` is the default there is no way of unsetting the value because `bool` parameters don't expect an argument.

Reply:
nit: can we call this `enable_limit_orders` like we do in the API? Then we can use the same flag for both components (and, practically speaking limit orders are pretty much disabled with this flag off).

Reply:
> as the value can still be set to false using the env variable

I think I vaguely remember that it used to not be the case with `StructOpt` which we used for a while. One downside I found is that I couldn't figure out how to use `--flag` command line options to turn off a `default_value = "true"` flag (without resorting to environment variables):

```rust
use clap::Parser;

#[derive(Debug, Parser)]
struct Arguments {
    #[clap(long, env, default_value = "true")]
    my_flag: bool,
}

fn main() {
    let args = Arguments::parse();
    println!("{args:?}");
}
```

```
% cargo run
Arguments { my_flag: true }

% cargo run -- --my-flag
Arguments { my_flag: true }

% cargo run -- --my-flag=false
error: The value 'false' was provided to '--my-flag' but it wasn't expecting any more values

Usage: clap-test --my-flag

For more information try '--help'

% MY_FLAG=false cargo run
Arguments { my_flag: false }
```

Not a big deal because we only use environment for configuring things in our deployments, but a little less ergonomic for local testing.